✨ Adding OneAD AD Subsystem

zonetw opened the pull request:
implements OneAD ad integration

Note: Please use browser's mobile simulator to preview our ads. (Our ad serving will not work in desktop environment)

zonetw commented:
signed CLA

zonetw commented:
@torch2424 @zhouyx Could you please help review this PR ?

torch2424 commented on ads/onead.md:
```diff
@@ -0,0 +1,22 @@
+# OneAD
```
Missing copyright on the top of the file

see: https://raw.githubusercontent.com/shopping24/amphtml/67cb7ea4b7ac75eabeb4605e1c41b9a7f0c8daef/ads/_ping_.md

zonetw replied:
@torch2424 
1. added copyright
2. Sorry for the empty content, I've updated the material.
3. I notice that the device you use is not mobile simulator.
Please use the mobile simulator, since we only show ad in mobile device.

Thanks a lot for your quick response :)

zonetw commented:
@torch2424 
Thanks a lot for your quick response :)

1. added copyright
2. Sorry for the empty content, I've updated the material.
3. I notice that the device you use is not mobile simulator.
Please use the mobile simulator, since we only show ad in mobile device.

torch2424 commented on ads/onead.js:
```diff
@@ -0,0 +1,64 @@
+ */
+function createAdUnit(win) {
+  const src = 'https://ad-specs.guoshipartners.com/static/js/onead-amp.min.js';
+  const js = document.createElement('script');
```
Oops! My apologies on missing this on the first review.

Could you use our helper function `loadScript` here?

https://github.com/ampproject/amphtml/blob/master/3p/3p.js#L100

And if you need to add any data to the script url, please be sure to use `encodeURIComponent`. E.g: https://github.com/ampproject/amphtml/blob/master/3p/github.js#L47

zonetw commented:
@torch2424 
I've changed to use `loadScript` and merge master again.
Please help me check again, thanks a lot !!

torch2424 commented:
Thanks for contributing! 😄 Everything looks great!

zonetw commented:
@torch2424 Thanks for helping review :)
